src/app/domain/case-viewer/components/timeline-panel/timeline-panel.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="govuk-grid-row">
+
+    <div class="govuk-grid-column-two-thirds">
```

Comment:
Please could we remove the whitespace / empty lines in this file. 

Possibly be a good idea to update your IDE to auto format against the tslint file in the repo ...